bedrock/firefox/views.py

```diff
@@ -281,6 +282,14 @@ def fx_home_redirect(request):
     return HttpResponseRedirect(reverse('firefox.new'))
 
 
+def choose(request):
+    if waffle.switch_is_active('tracking-protection-redirect'):
```

Comment:
Oh... nm. You switched it. I got confused b/c the name is now odd... hmm... `tracking-protection-redirect == on` means don't redirect. I'm fine w/ that, but perhaps confusing. Sorry for the churn. Are you cool w/ that or want to rename?